review-database/postgres/data-generation/reviews.generator.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const getRandomIntInclusive = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
+
+let id = 1;
+
+const generateReviews = (n, start, amount) => {
+  const write = fs.createWriteStream(`reviews${n}.data.csv`);
+  for (let i = start; i <= start + amount; i += 1) {
+    for (let j = 1; j <= getRandomIntInclusive(1, 5); j += 1) {
+      let result = `${id}
```

Comment:
I might consider defining all these values separately outside of your 'result' assignment, so that it's clearer what the meaning of each one is...e.g., reviewId = id++; reviewDate = new Date()..., and then return reviewId + reviewDate... . Not sure if this is actually better practice (or if it's worth implementing now), but it seems like it would better communicate how your data is set up.